hack/create-project/command.go

```diff
@@ -66,44 +72,37 @@ func (c *command) Run(ctx context.Context, metrics metrics.Metrics, logger *zap.
 	req := &environmentproto.CreateProjectRequest{
 		Command: &environmentproto.CreateProjectCommand{
 			Name:        *c.name,
-			UrlCode:     *c.name,
+			UrlCode:     strings.ToLower(strings.ReplaceAll(*c.urlCode, " ", "-")),
```

Comment:
I added this line to lower the string and replace whitespace for hyphens. Same thing while creating environments.